statistics/handle/handle.go

```diff
@@ -389,7 +389,13 @@ func (h *Handle) MergePartitionStats2GlobalStats(is infoschema.InfoSchema, physi
 		}
 
 		// Merge histogram
-		err = errors.Errorf("TODO: The merge function of the histogram structure has not been implemented yet")
+		globalStats.Hg[i], err = statistics.MergePartitionHist2GlobalHist(sc, allHg[i], 0)
```

Comment:
Can we explicitly set the number of histogram buckets (the third parameter of the MergePartitionHist2GlobalHist function)?